examples/oneapi/cpp/source/knn/knn_cls_brute_force_chebychev_distance_dense_batch.cpp

```diff
@@ -0,0 +1,49 @@
+/*******************************************************************************
```

Comment:
I propose delete new examples, move check with metrics to tests

Reply:
replaced with tests